data/ct/executive/Denise-Merrill-330c3bb4-3f34-40ee-8a6f-71cd84849d80.yml

```diff
@@ -7,17 +7,17 @@ image: https://portal.ct.gov/lib/sots/capitol2011to2015/headshot/dm_official_hea
 party:
 - name: Democratic
 roles:
-- end_date: '2021-12-31'
+- end_date: '2022-06-30'
   type: secretary of state
   jurisdiction: ocd-jurisdiction/country:us/state:ct/government
+offices:
+- classification: capitol
+  address: Capitol Office; PO Box 150470; Hartford, CT 06115-0470
+  voice: 860-509-6200
+  fax: 860-509-6209
 links:
```

Comment:
@bfossen-ce this is an executive who left on 06/30/2022, so I think they should be in retired.